playground/frontend/lib/modules/editor/components/pipeline_options_dropdown/pipeline_options_dropdown_body.dart

```diff
@@ -29,19 +29,16 @@ import 'package:playground/modules/editor/parsers/run_options_parser.dart';
 const kOptionsTabIndex = 0;
 const kRawTabIndex = 1;
 
-final kDefaultOption = [PipelineOptionController()];
-
 class PipelineOptionsDropdownBody extends StatefulWidget {
   final String pipelineOptions;
   final void Function(String) setPipelineOptions;
   final void Function() close;
 
-  const PipelineOptionsDropdownBody({
-    Key? key,
+  PipelineOptionsDropdownBody({
     required this.pipelineOptions,
     required this.setPipelineOptions,
     required this.close,
-  }) : super(key: key);
+  }) : super(key: ValueKey(pipelineOptions));
```

Comment:
So that the old state is dropped and options are re-parsed to new controllers if they change.